scorer.pl

```diff
@@ -31,10 +37,12 @@ BEGIN
 
   response_file: file with output of coreference system (SemEval format)
 
-  name: [optional] the name of the document to score. If name is not
+  -n name: [optional] the name of the document to score. If name is not
     given, all the documents in the dataset will be scored. If given
     name is "none" then all the documents are scored but only total
     results are shown.
+    
+  -a: [optional] flag to allow multiple tagging of an entity
```

Comment:
the phrase "multiple tagging" is not quite intuitive. maybe use "multiple cluster membership"
-a is not very intuitive. maybe use M for multiple cluster membership? maybe also create a long option --multiple-cluster-membership-allowed